34/javadoc/search.js

```diff
@@ -1,26 +1,26 @@
 /*
  * Copyright (c) 2015, 2020, Oracle and/or its affiliates. All rights reserved.
- * DO NOT ALTER OR REMOVE COPYRIGHT NOTICES OR THIS FILE HEADER.
+ * ORACLE PROPRIETARY/CONFIDENTIAL. Use is subject to license terms.
+ *
+ *
+ *
+ *
+ *
+ *
+ *
+ *
+ *
+ *
+ *
+ *
+ *
+ *
+ *
+ *
```

Comment:
Same as above

Reply:
Change reverted in https://github.com/apache/kafka-site/pull/519/commits/3b39b2a6b7c459ef9f53a3b5acf9ed1aedc39307